azure-iot-hub/azure/iot/hub/iothub_amqp_client.py

```diff
@@ -0,0 +1,83 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+import os
+import sys
+import base64
+import time
+import hashlib
+import hmac
+from uuid import uuid4
+import six.moves.urllib as urllib
+
+try:
+    from urllib import quote, quote_plus, urlencode  # Py2
+except Exception:
+    from urllib.parse import quote, quote_plus, urlencode
```

Comment:
you don't need this if you're using six above.  Just reference these with urllib.parse.quote, etc in the code.  Or keep the 'from urllib.parse import quote, quote_plus, urlencode' part of this.  After using six, the py2 block is dead because six makes the py2 library match the py3 signature.